tests/regression/src/bin/config_configure.rs

```diff
@@ -0,0 +1,29 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Hmm, generally I think I'd actually prefer to do more of the configuration directly rather than shelling out to such tiny functions.